### DISCUSSION TITLE
Release 62 BT goggles

### DISCUSSION THREAD

**Discussion by Never6** [chromiecraft/chromiecraft#5080] [Suggestions]
title pretty self explanitory
making the argument that its not blizzlike because the item released in BT in retail tbc is just disingenuous 
If you are gonna make that argument then nagrand arena should have a tornado, no ruins of lord, no skettis no ogri'la no netherwing, and mag and kael won't drop items to turn in because all those things also released during black temple in retail tbc
But that would be stupid
for reference my helm has 31agi 47stam and 63ap vs 62 goggles 48agi 28stam 76 ap and 1% hit and a meta and blue gem
so i have to wait until BT to get a +10ap upgrade of a level 68 blue
meanwhile deep thunder which is t4 bis for some classes has been out since 64

**Comment by balleny** [comment]
cross check with wotlk classic release schedule. this seems to be the new default applied

**Reply by Never6** [reply]
If going by TBC classic is the new way of progression then 64-69 shouldnt have even been a bracket, we just would go straight to 70
also every normal would be released by now
all of t4 would release at once instead of kara after some time
classic also had nagrand tornado for s1 arena  but i doubt this server is gonna have that
If you are going to go by classic  then actually be consistent with it

**Reply by balleny** [reply]
you have been playing long enough on CC to know that the brackets are there for the devs, bug fixers and triagers and the reporters to report and correct things.
progression is applied with content fixing in mind.

Your pseudo rant about this is contributing null to your initial topic of getting a better profession balance. Blizzard released the goggles in p2 on classic, and towards BT on retail. Blacksmith weapons were released in p1.

if you would want to see that changed, better bring reasonable arguments instead of whining like a little toddler

**Reply by Never6** [reply]
the point is that, if they  are going to go by tbc classic release then actually go all the way with it.
But its 3.3.5 not 2.4.3 it makes no sense to actually do that.
If you wanna act like a genuine cunt don't bother actually posting anything
its 3.3.5 the goggles should be out and if anyone with a brain actually looked at stats they would be released because they are barely better than 52

**Reply by Never6** [reply]
Maybe fix the massive core issues first before releasing tbc

**Reply by Never6** [reply]
> you have been playing long enough on CC to know that the brackets are there for the devs, bug fixers and triagers and the reporters to report and correct things. progression is applied with content fixing in mind.
> 
> Your pseudo rant about this is contributing null to your initial topic of getting a better profession balance. Blizzard released the goggles in p2 on classic, and towards BT on retail. Blacksmith weapons were released in p1.
> 
> if you would want to see that changed, better bring reasonable arguments instead of whining like a little toddler

literally a worthless human being

**Reply by Never6** [reply]
> you have been playing long enough on CC to know that the brackets are there for the devs, bug fixers and triagers and the reporters to report and correct things. progression is applied with content fixing in mind.
what does locking 62 goggles have anything to do with bug fixing

**Comment by Never6** [comment]
Fine i guess i close this.
Seems like people wanna follow classic release all the way even when it makes zero sense. Rather than making their game work
guess what server still has massive core bugs that still havent been fixed

**Comment by Jinteki01** [comment]
There is already a discussion about the goggles release here https://github.com/chromiecraft/chromiecraft/discussions/4831 and, as was pointed out there, other professions have crafted gear that is on par with T4 gear.
